test/spec_request.rb

```diff
@@ -214,7 +220,7 @@ class RackRequestTest < Minitest::Spec
     req.host_with_port.must_equal "example.org:9292"
 
     req = make_request \
-      Rack::MockRequest.env_for("/", "SERVER_NAME" => "example.org", "SERVER_PORT" => "")
```

Comment:
Likewise, I think we can remove this test.  `SERVER_PORT` is already required by SPEC to be present and not empty, and we don't need to test undefined behavior.

Reply:
Test was removed.